content/en/agent/proxy.md

```diff
@@ -174,8 +174,14 @@ resolvers my-dns
 # sending metrics (e.g. the value of "dd_url").
 frontend metrics-forwarder
```

Comment:
The `frontend metrics-forwarder` in the EU configuration sample has not been updated accordingly.